lib/p2p/Peer.ts

```diff
@@ -50,6 +50,8 @@ class Peer extends EventEmitter {
   public sentDisconnectionReason?: DisconnectionReason;
   public expectedNodePubKey?: string;
   public active = false; // added to peer list
+  /** Timer to schedularly call getNodes #402. */
```

Comment:
`schedularly` sounds like it should be a word, but I confirmed that it's not. How about just `Timer to periodically call GetNodes`?

Reply:
haha, my bad of course it's not.